containers/signing-service/Dockerfile.j2

```diff
@@ -0,0 +1,15 @@
+FROM debian:{{ debian_version }}
+
+COPY testing.pref                       /etc/apt/preferences.d/
+COPY install-dependencies.sh            /tmp/
+RUN  /tmp/install-dependencies.sh
+
+COPY signing-service/requirements.lock  /srv/http/signing_service/
+COPY install-signing-service.sh         /tmp/
+RUN  /tmp/install-signing-service.sh
+
+USER signingserviceuser
+COPY signing-service                    /srv/http/
```

Comment:
This is not a server app. Let's put it somewhere else. Let's use `/usr/lib/signing-service/`. It should be owned by root and read-only. Our user does not need to modify these files anyway.

Reply:
Done.